src/org/zaproxy/zap/extension/callback/ExtensionCallback.java

```diff
@@ -232,4 +251,8 @@ public boolean onHttpResponseReceived(HttpMessage msg) {
         }
 
     }
+
+    private void callBackReceived(String handler, HttpMessage httpMessage){
+        callbackPanel.addCallbackModel(new CallbackModel(handler, new Date(), httpMessage));
```

Comment:
This should be skipped when there's no view (in daemon mode it results in a NPE).